diff -Naur DB_File-1.831.orig/Makefile.PL DB_File-1.831/Makefile.PL
--- DB_File-1.831.orig/Makefile.PL	2014-04-28 12:37:24.468116624 +0400
+++ DB_File-1.831/Makefile.PL	2014-04-28 12:37:53.644115824 +0400
@@ -69,7 +69,7 @@
 	'depend'	=> { 'Makefile'          => 'config.in',
                              'version$(OBJ_EXT)' => 'version.c'},
 	'clean'		=> { FILES => 'constants.h constants.xs DB_File.pm.bak t/db-btree.t.bak t/db-hash.t.bak t/db-recno.t.bak t/pod.t.bak' },
-	'macro'		=> { my_files => "@files" },
+#	'macro'		=> { my_files => "@files" },
         'dist'          => { COMPRESS => 'gzip', SUFFIX => 'gz',
 			     DIST_DEFAULT => 'MyDoubleCheck tardist'},    
 	);
